workitem/link/link_repository.go

```diff
@@ -428,7 +428,7 @@ func (r *GormWorkItemLinkRepository) ListWorkItemChildren(ctx context.Context, p
 	where := fmt.Sprintf(`
 	id in (
 		SELECT target_id FROM %s
-		WHERE source_id = $1 AND link_type_id = $2
+		WHERE source_id = $1 AND link_type_id = $2 AND deleted_at IS NULL
```

Comment:
@kwk We are not using the `EXISTS` sub-query expression here. But I can see it's used other places in the same package. Is that required?

Reply:
@baijum Sure, why not. I think `EXISTS` could be used here as well.

Reply:
https://www.zimmi.cz/posts/2015/postgresql-in-vs-exists/

Reply:
@baijum this query is wrong, you are selecting just 1 row.

Reply:
@DhritiShikhar yeah. that query seems to be not working.